src/app/modules/webhook/webhook.service.ts

```diff
@@ -156,6 +156,8 @@ export const sendWebhook = (
                 signature,
               }),
             },
+            decompress: false,
+            maxContentLength: 10000,
```

Comment:
Might want to move this `10000` to a constant under `webhook/webhook.constants.ts`

(how was this 10000 derived?)

Reply:
Will move the configuration as suggested.

The webhook response is mostly expected to be empty. A quick check on the recent webhook responses in our prod analytics DB also indicates that the response is empty.

Given that the response of the webhook call is not used and does not impact our partner's processing of the call, I have revised the limit to 1MB, which should accommodate most webhook responses. What are your thoughts?

Reply:
I see. Thanks for checking! I think this arrangement makes sense.